File: Problemafelvetes1_java/Member.java
public class Member {
    private String memberId;
    private String name;
    private String email;

    public Member(String memberId, String name, String email) {
        this.memberId = memberId;
        this.name = name;
        this.email = email;
    }

    @Override
    public String toString() {
        return "Member{" +
               "memberId='" + this.memberId + '\'' +
               ", name='" + this.name + '\'' +
               ", email='" + this.email + '\'' +
               '}';
    }
}

File: Problemafelvetes1/Classes/Member.py

class Member:
    def __init__(self, name, member_id, email):
        self._name = name
        self._member_id = member_id
        self._email = email


    email = property(get_email, set_email)
    @property
    def email(self):
        return self._email
    @email.setter
    def email(self, email):
        self._email = email

